counterparty-core/counterpartycore/lib/blocks.py

```diff
@@ -238,13 +252,19 @@ def parse_tx(db, tx):
                 util.CURRENT_TX_HASH = None
                 return False
 
+            # if attach or detach we move assets after parsing
+            if util.enabled("spend_utxo_to_detach") and message_type_id in [attach.ID, detach.ID]:
```

Comment:
There's likely no need to call "move.move_assets(db, tx)" if it’s a detach transaction.

`if util.enabled("spend_utxo_to_detach") and message_type_id in [attach.ID, detach.ID]:
                move.move_assets(db, tx)`

Reply:
yeah right, no need anymore now that detach can't fail! will fix that.

Reply:
done here https://github.com/CounterpartyXCP/counterparty-core/pull/2626/commits/d65072563ff8c56949ba555ad13227656c41bd7c